### ISSUE TITLE
Midpoint Review - Design Feedback - Transition Experience, tbd, V1 Transition Experience

### ISSUE THREAD

**Issue by allison0034** [department-of-veterans-affairs/va.gov-team#97164]
## Next Steps for the VFS team

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

- This recently went thur DSC for experimental design. I am documenting the result of that meeting. Summary from DSC meeting:
   - Not a candidate to bring to DS for now.
   - Would prefer you stick to using the tags b/c it is something we have
   - I would not make it a candidate to bring to DS for now.
   - You can use in your application, but if we do add the warning one Jasmine shared, we would ask you to go back and update it.
   - Will use this for version 2, then for version 3, will work with Jasmine to implement colored tags.

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [x] Must: ~~Currently our guidance says that all forms must include the [penalty notice component](https://design.va.gov/components/form/penalty-notice). Add the component to your review page. Your application is slightly different, if you can get legal to agree it is not necessary feel free to comment on this ticket.~~ After [speaking with Matt](https://dsva.slack.com/archives/C06C7RS27GD/p1730907671846339), the penalty notice is not necessary however you must keep the privacy policy.
- [x] Must: Look at the currently [experimental design tickets around filtering](https://github.com/orgs/department-of-veterans-affairs/projects/946/views/3?filterQuery=filter). We are very close to having an established pattern, specifically, [MHV secure messaging](https://github.com/orgs/department-of-veterans-affairs/projects/946/views/3?filterQuery=filter&pane=issue&itemId=29995760&issue=department-of-veterans-affairs%7Cvets-design-system-documentation%7C1582). Reach out to those teams and see if you can align closer to how their filtering works.
- [ ] Should: I am fearful users may be confused if they click "Show every benefit" and navigate away from the benefits they qualify for. They may not realize or might forget that they do not qualify for all the benefits listed. You could add a radio button under the filter for all benefits vs the ones I qualify for. If you show all benefits you may want to consider adding some text or cue in the benefits cards that they do or do not qualify. Just trying to be super clear to the user about what they do or do not qualify for.
- [x] Should: We have noticed a lot of alert fatigue on VA.gov. When a user does not have results there is a warning alert. Typically [warning alerts](https://design.va.gov/components/alert/#warning-alert) are used when there are negative consequences, or when something has gone wrong. I don't think no results warrants a warning style. In fact, I would consider removing any alert styling and just have text especially since you already have an alert on the page.
- [ ] Should: It seems the time sensitive benefits would be most important but they are currently mixed in with the rest of the results. Consider adding a way to sort by or filter by time sensitive or list time sensitive at the top be default. 

## Governance team actions
- [ ] Format feedback as individual tasks (check boxes)
- [ ] Assign this ticket to the VFS team member that opened the Slack request
- [ ] Add the VFS team product label
- [ ] Add the VFS team the feature label (if applicable)
- [ ] Add the touchpoint labels
- [ ] Add the practice area labels
- [ ] Add the Collaboration Cycle initiative milestone

**Comment by nkontrabecki** [comment]
@allison0034  Good Morning. Thank you for the detailed feedback. Regarding the [penalty notice component](https://design.va.gov/components/form/penalty-notice) feedback, please advise who the legal POC is who we should work with.

**Comment by allison0034** [comment]
@nkontrabecki I am going to reeach out to Matt Dingee today and get clarification.

**Comment by nkontrabecki** [comment]
> [@nkontrabecki](https://github.com/nkontrabecki) I am going to reeach out to Matt Dingee today and get clarification.

Thank you. Happy to find a time to chat through this if needed.

**Comment by nkontrabecki** [comment]
Hi @allison0034. Related to the second MUST feedback comment, please clarify - are we expected to actually overhaul our filters before release? The MHV filtering pattern looks complex and would require us to add [text filtering capabilities](https://github.com/orgs/department-of-veterans-affairs/projects/946/views/3?filterQuery=filter&pane=issue&itemId=29995760&issue=department-of-veterans-affairs%7Cvets-design-system-documentation%7C1582)

Our team will connect with the MHV messaging team about their filtering system, especially since it's about to become part of the VADS pattern library. It looks like Ben Brasso is on the team - we chatted with him about his appointment scheduling product back in June.

Once we have received your response we will need to consider the level of development effort to re-do the filters and launch timing.

**Comment by allison0034** [comment]
@nkontrabecki no, these tickets are experimental so they are not an established pattern yet. You should look at them and become familiar, at some point this will be a pattern in the design system.

**Comment by nkontrabecki** [comment]
Thank you @allison0034  The team will review the experimental design tickets around filtering.

**Comment by artsymartha68** [comment]
Team, I happened to be trying out the tool today and noticed that there are some hard-coded line breaks throughout. Can we add this as a MUST fix to remove these so that the text can flow well, regardless of zoom level? ![Image](https://github.com/user-attachments/assets/af79e2e2-cf06-4d3f-9e49-d8449ad9b88b)

**Comment by allison0034** [comment]
@briandeconinck can you add that to your ticket? FYI @nkontrabecki

**Comment by briandeconinck** [comment]
This was already noted by the team in their accessibility artifact for their upcoming staging review.

**Comment by nkontrabecki** [comment]
For this piece of feedback:

"Should: We have noticed a lot of alert fatigue on VA.gov. When a user does not have results there is a warning alert. Typically [warning alerts](https://design.va.gov/components/alert/#warning-alert) are used when there are negative consequences, or when something has gone wrong. I don't think no results warrants a warning style. In fact, I would consider removing any alert styling and just have text especially since you already have an alert on the page."

we updated the product to make Q1 a required question which ensures the user is never presented an empty state on the Results page.
